tst/EnergyPlus/unit/AirflowNetworkBalanceManager.unit.cc

```diff
@@ -20013,4 +20013,134 @@ std::string const idf_objects = delimited_string({
     EXPECT_EQ(AirflowNetwork::DisSysCompCVFData(1).AirLoopNum,1);
 
 }
+TEST_F(EnergyPlusFixture, AirflowNetwork_TestZoneVentingAirBoundary)
```

Comment:
New unit test to verify ventilation control and availability schedule handling for air boundary and normal surfaces.